docs/query-building.md

```diff
@@ -93,3 +93,60 @@ var_dump($result->getResults());
 
 You can abstract parts of your query to separate classes and re-use them. More about it in the [re-useable fragments](re-useable-fragments.md)
 documentation.
+
+## Pagination
+
+You can use a pagination trait in your query like this:
+
+```php
+use ElasticSearcher\Abstracts\AbstractQuery;
+use ElasticSearcher\Fragments\Traits\PaginatedTrait;
+
+class MoviesYouMightLikeQuery extends AbstractQuery
+{
+  use PaginatedTrait;
+
+  public function setup()
+  {
+    $this->searchIn('suggestions', 'movies');
+
+    // Page 2 with 20 results per page.
+    $this->paginate(2, 20);
```

Comment:
not sure if you would use this in here? I believe I'd call it as a user of `MoviesYouMightLikeQuery `